capture.go

```diff
@@ -0,0 +1,140 @@
+// Package command provides an organized way of running a *exec.Cmd and generates
+// a Capture that contains all commonly-accessed information in the Cmd that allows
+// for repeated execution.
+package command
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+)
+
+// Capture represents the state for a run of a command.
+type Capture struct {
+	// Name is the name to run.
+	//
+	// This is the only field that must be set to a non-zero
+	// value. If Path is relative, it is evaluated relative
+	// to Dir.
+	Name string `json:"name"`
+
+	// Args holds name line arguments, excluding the name as Args[0].
+	// This differs in behavior from the exec.Cmd type.
+	Args []string `json:"args,omitempty"`
+
+	// Dir specifies the working directory of the name.
+	// If Dir is the empty string, Comand runs the name in the
+	// calling process's current directory.
+	Dir string `json:"working_dir,omitempty"`
+
+	// Env specifies the environment of the process.
+	// Each entry is of the form "key=value".
+	// If Env is nil, the new process uses the current process's
+	// environment.
+	// If Env contains duplicate environment keys, only the last
+	// value in the slice for each duplicate key is used.
+	// As a special case on Windows, SYSTEMROOT is always added if
```

Comment:
is this part of the built in behavior that you're just calling out for clarity? We're not explicitly doing this ourselves right?

Reply:
Yes, the behavior of `exec.Cmd` is surprising sometimes, and I wanted to copy its docs out because it's the underlying implementation.